util/jsonrpc-types/src/blockchain.rs

```diff
@@ -637,16 +637,20 @@ pub struct Header {
     ///
     /// It is all zeros when `proposals` is empty, or the hash on all the bytes concatenated together.
     pub proposals_hash: H256,
-    // TODO ckb2021 Returns the extra hash as uncles hash directly since no extension now.
-    // The hash on `uncles` and extension in the block body.
-    //
-    // The uncles hash is all zeros when `uncles` is empty, or the hash on all the uncle header hashes concatenated together.
-    // The extension hash is the hash of the extension.
-    // The extra hash is the hash on uncles hash and extension hash concatenated together.
-    /// The hash on `uncles` in the block body.
+    // TODO ckb2021 Update the rfc number and fix the link, after the proposal is merged.
+    /// The hash on `uncles` and extension in the block body.
     ///
-    /// It is all zeros when `uncles` is empty, or the hash on all the uncle header hashes concatenated together.
-    pub uncles_hash: H256,
+    /// The uncles hash is all zeros when `uncles` is empty, or the hash on all the uncle header hashes concatenated together.
+    /// The extension hash is the hash of the extension.
+    /// The extra hash is the hash on uncles hash and extension hash concatenated together.
+    ///
+    /// # Notice
+    ///
+    /// This field is renamed from `uncles_hash` since 0.44.0.
+    /// More details can be found in [CKB RFC xxxx].
+    ///
+    /// [CKB RFC xxxx]: https://github.com/nervosnetwork/rfcs/tree/master/rfcs/xxxx-rfc-title
```

Comment:
`rfcs/0224-variable-length-header-field/0224-variable-length-header-field.md`

Reply:
I comment the "TODO ckb2021" above, and there are other links and comments require updating.

We can update all links and all comments at one commit later, after all proposals frozen.